doc/createrepo_c.8

```diff
@@ -90,6 +90,10 @@ Optional output directory.
 .SS \-S \-\-skip\-symlinks
 .sp
 Ignore symlinks of packages.
+.SS \--split
+.sp
+Run in split media mode. Rather than pass a single directory, take a set of
+directories corresponding to different volumes in a media set.
```

Comment:
It's good that you added it to the man page, but I would prefer that this be also in the command parser code because occasionally the man pages are regenerated from the source code. See the utils directory for the scripts that can do this.

I prefer not having to regenerate the man pages, but I'd rather not run the risk of losing good descriptions due to someone running it.